onnxruntime/test/providers/cpu/math/gemm_test.cc

```diff
@@ -641,6 +641,47 @@ TYPED_TEST(GemmOpTypedTests, GemmEmptyTensor) {
       .Config(run_with_tunable_op)
       .RunWithConfig();
 }
+
+TYPED_TEST(GemmOpTypedTests, ZeroKWithBias) {
```

Comment:
Should we have a test case for M==0 && N==0 && K==0 ?

Reply:
The code tests for M or N earlier than for K.